tests/plugins/pastetools/filter/parsestyles.js

```diff
@@ -6,6 +6,11 @@
 ( function() {
 	'use strict';
 
+	// (#3713)
+	if ( CKEDITOR.env.edge && CKEDITOR.env.version < 80 ) {
+		bender.ignore();
+	}
+
```

Comment:
Not sure if we would ever need to run this tests on older Edge versions (\`edgeHTML\`) :thinking: Maybe we should just leave it and also close #3713 as Edge migrated to Chromium.